yarn.lock

```diff
@@ -3218,6 +3213,11 @@
   resolved "https://registry.yarnpkg.com/@types/less/-/less-3.0.1.tgz#625694093c72f8356c4042754e222407e50d6b08"
   integrity sha512-dBp05MtWN/w1fGVjj5LVrDw6VrdYllpWczbUkCsrzBj08IHsSyRLOFvUrCFqZFVR+nsqkrRLNg6oOlvqMLPaSA==
 
+"@types/lodash@^4.14.117":
```

Comment:
why do we suddenly have `@types/lodash` bc of this? does yoco even really on lodash or did you do an update as well or what :thinking:

Reply:
Those most likely come from `yoco`s `devDependencies` for the new build process:
<https://github.com/3YOURMIND/kotti/pull/259/files#diff-85807792aef76b0bddd8749ae410dd0aR19-R20>